src/bp-templates/bp-nouveau/js/buddypress-activity-post-form.js

```diff
@@ -1568,6 +1568,10 @@ window.bp = window.bp || {};
 						return false;
 					}
 
+					if ( url.includes( window.location.hostname ) ) {
```

Comment:
@krunal-buddyboss Please update the logic here, If the user wants to share a site blog here then, It will not embed.

Reply:
@KartikSuthar 
I have applied changes as we have discussed on slack.
FYI @sourabhmatolia @chetansatasiya @mohit-buddyboss